app/Controllers/authController.php

```diff
@@ -120,7 +120,7 @@ public function formLoginAction(): void {
 
 			if ($nonce === '') {
 				Minz_Log::warning("Invalid session during login for user={$username}, nonce={$nonce}");
-				header('HTTP/1.1 403 Forbidden');
+				header(FreshRSS_HttpResponseCode::descriptionFromCode(FreshRSS_HttpResponseCode::HTTP_403_FORBIDDEN));
```

Comment:
This does not look nicer to me. I can better like the string, which is a well-known standard, but even with the code version, I think we can do it better with some `__toString()` methods

Reply:
I've improved the code to make it less wordy